plugins/AIActivity/FeatureFlags/AIActivity.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+declare(strict_types=1);
+
+namespace Piwik\Plugins\AIActivity\FeatureFlags;
+
+use Piwik\Plugins\FeatureFlags\FeatureFlagInterface;
+
+class AIActivity implements FeatureFlagInterface
```

Comment:
I actually don't think we need a feature flag for this at all. As long as the plugin is disabled by default no one would see it automatically. Also it's not a half baked feature that doesn't work properly. The report provided by the plugin will be usable and valuable, so no need to additionally hide it behind a feature flag - unless it is planned to extend this a lot more and release everything with a big bang